### DISCUSSION TITLE
What options are available for walletConnect Connectors?

### DISCUSSION THREAD

**Discussion by madhawk1296** [wevm/wagmi#1403] [Q&A]
What options are available for walletConnect Connectors?

**Comment by tmm** [comment]
Looks like WalletConnect removed the docs for this, but you can find the options here: https://github.com/WalletConnect/walletconnect-monorepo/blob/1.8.0/packages/helpers/types/index.d.ts#L300-L314

**Reply by QuintonC** [reply]
I was actively looking at that this morning as well. 

I noticed that when providing custom wallet options for `mobileLinks` the icons aren't showing up for the wallets.

Is there any active work on migrating to WalletConnect 2.0?

**Reply by tmm** [reply]
Yes, the wagmi core team and WalletConnect team are collaborating on https://github.com/wagmi-dev/wagmi/pull/1208
